<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>florian.sprich@gmail.com</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>F1s3FR94?!</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gegner_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SC Paderborn</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gegner_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TSG 1899 Hoffenheim</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gegner_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fortuna DÃ¼sseldorf</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gegner_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1. FC Union Berlin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>400</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email_receiver">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>florian.sprich@gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_receiver">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email receiver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hideBrowser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide Browser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_receiver_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(komma separated)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
